### DISCUSSION TITLE
Compatible with 1.20.6

### DISCUSSION THREAD

**Discussion by aixcyi** [P3pp3rF1y/SophisticatedBackpacks#1086] [General]
With the official release of 1.21, the 1.20.x seems to have stabilized. Should compatibility with 1.20.6 be put on the plan?

**Comment by P3pp3rF1y** [comment]
I have no plan for 1.20.6, 1.20.6 is going to be that version that was very quickly replaced with a new one and most of the mods won't be ported to it. So I am also skipping it in favor of 1.21 and I will continue supporting 1.20.1 for some time as a long running 1.20 version (the other 1.20 versions were really just 1.21 unofficial pre-releases.
